src/Host.cpp

```diff
@@ -369,6 +369,28 @@ void Host::resetProfile()
     qDebug()<<"resetProfile() DONE";
 }
 
+// Saves profile to disk - does not save items dirty in the editor, however.
+// returns true if successful or false+error message if not
+std::tuple<bool, QString, QString, QString, QString> Host::saveProfile()
```

Comment:
I knew I had something, see `TLuaInterpreter::startLogging` that produces an internationalized message for display to a user - oh, no, it __doesn't__ any-more, once #941 goes through... :disappointed:

That is not such a useful example after all. :thinking: